README.md

```diff
@@ -13,7 +13,10 @@
 
```

Comment:
Can you change the Slack link above to this instead?
https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA

Reply:
Yep will change it in both places